README.md

```diff
@@ -1 +1,29 @@
-# team-culture
\ No newline at end of file
+# Code of Conduct
+
+## Diversity
+We accept others, no matter the gender, nationality, culture, religion.
+
+## Respect
+We all have different backgrounds, beliefs and points of view.
+Treat others as you wish to be treated.
+
+## Trust and Cooperation
+We are a team, don't be afraid to be judged. We support each other.
+
+## Autonomy
+You are the owner of your task, you choose how to do your tasks , when to ask for help, and when to pull the plug on a discussion that you asked for.
```

Comment:
```suggestion
We are the owner of our tasks, and we choose how to do our tasks, when to ask for help, and when to pull the plug on a discussion that we asked for.
```

Reply:
Why we? `We are the owner` doesn't make much sense. What do you want to say here?

Reply:
I think You tense is just wrong. I know my suggestion also doesn't fix it completely. But something is wrong here. The sentence has to be more elegant. It is rude right now.